plugins/woocommerce/src/Admin/Features/ProductBlockEditor/ProductTemplates/SimpleProductTemplate.php

```diff
@@ -0,0 +1,773 @@
+					),
+				],
+			]
+		);
+	}
+
+	/**
+	 * Adds the shipping group blocks to the template.
+	 */
+	private function add_shipping_group_blocks() {
+		$shipping_group = $this->add_group(
+			[
+				'id'         => 'shipping',
+				'order'      => 40,
+				'attributes' => [
+					'id'    => 'shipping',
+					'title' => __( 'Shipping', 'woocommerce' ),
+				],
+			]
+		);
+		$shipping_group->add_block(
+			[
+				'id'         => 'product_variation_notice_shipping_tab',
+				'blockName'  => 'woocommerce/product-has-variations-notice',
+				'attributes' => [
+					'id'         => 'wc-product-notice-has-options',
+					'content'    => __( 'This product has options, such as size or color. You can now manage each variation\'s price and other details individually.', 'woocommerce' ),
+					'buttonText' => __( 'Go to Variations', 'woocommerce' ),
+					'type'       => 'info',
+				],
+			]
+		);
+		// Product Pricing Section.
+		$product_fee_and_dimensions_section = $shipping_group->add_section(
+			[
+				'id'         => 'product-fee-and-dimensions-section',
+				'attributes' => [
+					'title'       => __( 'Fees & dimensions', 'woocommerce' ),
+					'description' => sprintf(
+					/* translators: %1$s: How to get started? link opening tag. %2$s: How to get started? link closing tag.*/
+						__( 'Set up shipping costs and enter dimensions used for accurate rate calculations. %1$sHow to get started?%2$s.', 'woocommerce' ),
+						'<a href="https://woocommerce.com/posts/how-to-calculate-shipping-costs-for-your-woocommerce-store/" target="_blank" rel="noreferrer">',
+						'</a>'
+					),
+				],
+			]
+		);
+		$product_fee_and_dimensions_section->add_block(
+			[
+				'id'        => 'product-shipping-class',
+				'blockName' => 'woocommerce/product-shipping-class-field',
+			]
+		);
+		$product_fee_and_dimensions_section->add_block(
+			[
+				'id'        => 'product-shipping-dimensions',
+				'blockName' => 'woocommerce/product-shipping-dimensions-fields',
+			]
+		);
+	}
+
+	/**
+	 * Adds the variation group blocks to the template.
+	 */
+	private function add_variation_group_blocks() {
+		$variation_group  = $this->add_group(
+			[
+				'id'         => 'variations',
+				'order'      => 50,
+				'attributes' => [
+					'id'    => 'variations',
+					'title' => __( 'Variations', 'woocommerce' ),
+				],
+			]
+		);
+		$variation_fields = $variation_group->add_block(
+			[
+				'id'         => 'product_variation-field-group',
+				'blockName'  => 'woocommerce/product-variations-fields',
+				'attributes' => [
+					'description' => sprintf(
+					/* translators: %1$s: Sell your product in multiple variations like size or color. strong opening tag. %2$s: Sell your product in multiple variations like size or color. strong closing tag.*/
+						__( '%1$sSell your product in multiple variations like size or color.%2$s Get started by adding options for the buyers to choose on the product page.', 'woocommerce' ),
+						'<strong>',
+						'</strong>'
+					),
```

Comment:
Any reason why `sprintf` was used instead of `createInterpolateElement`?

Reply:
Similar to the above `sprintf` comment.